cassava.cabal

```diff
@@ -65,12 +65,13 @@ Library
 
   Build-depends:
     array >= 0.4 && < 0.6,
-    attoparsec >= 0.10.2 && < 0.14,
+    attoparsec >= 0.11.3.0 && < 0.14,
     base >= 4.5 && < 5,
     bytestring >= 0.9.2 && < 0.11,
     containers >= 0.4.2 && < 0.6,
     deepseq >= 1.1 && < 1.5,
     hashable < 1.3,
+    scientific >= 0.3.4.7,
```

Comment:
I left it off originally as I only import it to get the actual data type itself, and I figured it was unlikely to move from `Data.Scientific.Scientific`.